sdk/transform/functions.go

```diff
@@ -25,7 +26,12 @@ func customFunctions() template.FuncMap {
 		"phoneIso3166": phoneIso3166,
 		"array_contains": func(a []interface{}, b string) bool {
```

Comment:
Why do we need this `array_contains` function at all? Sprig includes a `has` function to check if a list contains a given value (https://masterminds.github.io/sprig/lists.html).

Reply:
I guess we need to keep it for backward compatibility but we shouldn't be using it in any of our templates.